### PR TITLE
Clarify installation for VSCode

### PR DESCRIPTION
**Pull request by cartermp**
Remove Ionide from the places where I mention it without defining it.

### CODE REVIEW

File: docs/fsharp/get-started/install-fsharp.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ After the install completes, choose "Start Visual Studio". You can also launch i
 
 ## Install F# with Visual Studio Code
 
-You must have [git installed](https://git-scm.com/download) and available on your PATH to make use of project templates in Ionide. You can verify that it is installed correctly by typing `git --version` at a command prompt and pressing **Enter**.
+You must have [git installed](https://git-scm.com/download) and available on your PATH to make use of project templates. You can verify that it is installed correctly by typing `git --version` at a command prompt and pressing **Enter**.
 
 ### [macOS](#tab/macos)
 
-Ionide uses [Mono](http://www.mono-project.com). The easiest way to install Mono on macOS is via Homebrew. Simply type the following into your terminal:
+[Mono](http://www.mono-project.com) is used for [F# Interactive](https://docs.microsoft.com/dotnet/fsharp/tutorials/fsharp-interactive/) support. The easiest way to install Mono on macOS is via Homebrew. Simply type the following into your terminal:
```
Comment (mairaw) [inline docs/fsharp/get-started/install-fsharp.md]: relative link for build time validation and offline books

File: docs/fsharp/get-started/install-fsharp.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
 
 ### [Windows](#tab/windows)
 
-If you're on Windows, you must [install Visual Studio with F# support](#install-f-with-visual-studio). This installs all the necessary components to write, compile, and execute F# code.
+[Install Visual Studio with F# support](#install-f-with-visual-studio) if you're on Windows. This installs all the necessary components to write, compile, and execute F# code.
```
Comment (mairaw) [inline docs/fsharp/get-started/install-fsharp.md]: conditional sentence should come first

also probably would look better if the verb Install was out of the link

File: docs/fsharp/get-started/install-fsharp.md [hunk 3]
```diff
@@ -24,11 +24,11 @@ After the install completes, choose "Start Visual Studio". You can also launch i
```
Comment (mairaw) [inline docs/fsharp/get-started/install-fsharp.md]: do these changes warrant a ms.date change?

**Review by JRAlexander — APPROVED** [review]
LGTM.

File: docs/fsharp/get-started/install-fsharp.md [hunk 4]
```diff
@@ -49,7 +49,7 @@ You must also install the [.NET Core SDK](https://www.microsoft.com/net/download
 
 ### [Windows](#tab/windows)
 
-If you're on Windows, you must [install Visual Studio with F# support](#install-f-with-visual-studio). This installs all the necessary components to write, compile, and execute F# code.
+[Install Visual Studio with F# support](#install-f-with-visual-studio) if you're on Windows. This installs all the necessary components to write, compile, and execute F# code.
 
 You must also install the [.NET Core SDK](https://www.microsoft.com/net/download/).
```
Comment (mairaw) [inline docs/fsharp/get-started/install-fsharp.md]: You must also install -> Also install

**Pull request merged**